### PR TITLE
Issue #22 - Add Code to generate a full profile (US Only)(male or female).

### PR DESCRIPTION
**Pull request by p4tin**

### CODE REVIEW

**Comment by Pallinder** [comment]
Great. I need some documentation over this feature added to the readme as well before a merge.

**Comment by p4tin** [comment]
ok let me see what I can do..

**Comment by p4tin** [comment]
I hope that is what you wanted!!

File: random_data.go [hunk 1]
```diff
@@ -78,6 +80,24 @@ func randomFrom(source []string) string {
 	return source[seedAndReturnRandom(len(source))]
 }
 
+// Returns a random title, gender decides the gender of the name
+func Title(gender int) string {
```
Comment (Pallinder) [inline random_data.go]: Let's add some documentation for this as well.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# idea
```
Comment (Pallinder) [inline .gitignore]: What are these?

File: README.md [hunk 3]
```diff
@@ -133,6 +135,10 @@ func main() {
 	// Print full date like Thursday 22 Aug 2016
 	fmt.Println(randomdata.FullDate())
 
+	// Get a full profile and use the data in it
```
Comment (Pallinder) [inline README.md]: Would probably want this to say
"Get a complete and randomised profile of data generally used for users"

File: random_data.go (hunk 1 above)
Comment (p4tin) [inline random_data.go]: Done

**Pull request merged**